legacy/firmware/fsm_msg_common.h

```diff
@@ -75,7 +75,6 @@ bool get_features(Features *resp) {
   resp->capabilities[1] = Capability_Capability_Bitcoin_like;
   resp->capabilities[2] = Capability_Capability_Crypto;
   resp->capabilities[3] = Capability_Capability_Ethereum;
-  resp->capabilities[4] = Capability_Capability_Lisk;
```

Comment:
It seems to me we should reduce the capability count by one and shift the indices of NEM, Stellar and U2F.

Reply:
9c0a16a